packages/algoliasearch-helper/src/SearchResults/index.js

```diff
@@ -231,7 +231,7 @@ function findMatchingHierarchicalFacetFromAttributeName(
 }
  **/
 function SearchResults(state, results, options) {
-  var mainSubResponse = results[0];
+  var mainSubResponse = results[0] || {};
```

Comment:
Why should the first response be defaulted?

Reply:
this was needed for a previous iteration where it was possible no request was sent if there are no parameters. I reverted that, but this change stayed, as it allows the helper to be a bit more flexible and to be able to instantiate an empty SearchResults if needed